skimage_tomo.tex

```diff
@@ -572,42 +572,40 @@ \section*{Image processing capabilities}
 \label{fig:tomo_gallery}}
 \end{figure*}
 
-\texttt{scikit-image} offers all the classical operations of image
-processing, such as exposure and color adjustment, filtering,
-segmentation, features extraction, geometric transformations or
-measurements of region characteristics. For these
-different categories, the package includes both standard generic
-operations and more advanced algorithms. A sample of
-\texttt{scikit-image} capabilities is given in
+\texttt{scikit-image} offers most classical image processing operations,
+such as exposure and color adjustment, filtering,
+segmentation, feature extraction, geometric transformations, and
+measurements of region characteristics. In addition, some advanced
+algorithms are also implemented, a selection of which is illustrated in
```

Comment:
Fig. 3 shows examples with both classical operations (labeling connected components) and more advanced operations. But it's true that I selected mostly advanced operations for this figure... showing off a bit, I think :-). 

I'm going to merge, but maybe I'll rephrase a bit this part.